helm-chart/templates/route-smtp.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.ports.smtp }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: "smtp"
```

To be consinstent with the other route:

```suggestion
  name: "smtp--{{ include "maildev.fullname" . }}"
```